libs/platform/kernel/ebpf_platform_kernel.c

```diff
@@ -9,6 +9,8 @@
 
 #include <ntstrsafe.h>
 
+bool ebpf_fuzzing_enabled = false;
```

Comment:
can this be in #if CHK such that this cannot be used in retail bits by mistake?

Reply:
This only exists to suppress asserts etc that fire when fuzzing.